roles/nvidia-dgx-firmware/vars/main.yml

```diff
@@ -13,10 +17,21 @@ firmware_container_src_dir: "{{ inventory_dir }}/containers/dgx-firmware"
 # default log directory is set to the location of your inventory file (./deepops/config/)
 local_log_directory: "{{ inventory_dir }}"
 
-# Remote directory to save misc. temporary logs to
-nv_diag_dir: "/tmp/nv-diag"
+# Remote directory to save misc. temporary files to
+fw_dir: "/tmp/nvfw"
 
 # The OS/mgmt interface on the server
 nv_mgmt_interface: enp1s0f0 # DGX-1
 # nv_mgmt_interface: enp134s0f0 # DGX-2
 # nv_mgmt_interface: enp2s0f1 # DGX-Station
+
```

Comment:
Ditto- Can we remove this? I can't find where it's used

Reply:
This is used to generate a CSV that maps (hostnames, macs, IPs)L

https://github.com/NVIDIA/deepops/blob/master/roles/nvidia-dgx-diag/tasks/get-data.yml